examples/repository_with_combine.rb

```diff
@@ -15,25 +22,34 @@ def call(dataset)
     uri.query = URI.encode_www_form(dataset.params)
 
     http = Net::HTTP.new(uri.host, uri.port)
-    request_klass = Net::HTTP.const_get(Inflecto.classify(dataset.request_method))
+    request_klass = Net::HTTP.const_get(ROM::Inflector.classify(dataset.request_method))
 
     request = request_klass.new(uri.request_uri)
     dataset.headers.each_with_object(request) do |(header, value), request|
       request[header.to_s] = value
     end
 
+    puts uri.request_uri
     response = http.request(request)
   end
 end
 
 class ResponseHandler
   def call(response, dataset)
     if %i(post put patch).include?(dataset.request_method)
-      JSON.parse(response.body, symbolize_names: true)
+      bleh(JSON.parse(response.body, symbolize_names: true))
     else
-      Array([JSON.parse(response.body, symbolize_names: true)]).flatten
+      Array([JSON.parse(response.body, symbolize_names: true)]).flatten.map(&method(:bleh))
     end
   end
+
+  def bleh(item)
```

Comment:
attribute aliases do not seem to work properly with rom-http for some reason, so this is a hack

Reply:
@ianks why is this needed?